agent/exploits/webexploit.py

```diff
@@ -120,7 +120,8 @@ def _create_vulnerability(
             short_description=self.metadata.description,
             description=self.metadata.description,
             references={
-                "nvd.nist.gov": f"https://nvd.nist.gov/vuln/detail/{self.metadata.reference}",
+                f"nvd.nist.gov/{cve}": f"https://nvd.nist.gov/vuln/detail/{cve}"
+                for cve in self.metadata.reference
             },
```

Comment:
You need to make sure it's a list not string

Reply:
Good catch.

Reply:
+1